public/app/features/dashboard/dashgrid/DashboardEmpty.tsx

```diff
@@ -0,0 +1,221 @@
+            markdowns and other widgets.
+          </div>
+          <Button
+            size="lg"
+            icon="plus"
+            aria-label="Add new panel"
+            onClick={() => {
+              reportInteraction('Create new panel');
+              onCreateNewPanel();
+            }}
+            disabled={!canCreate}
+          >
+            Add visualization
+          </Button>
+        </div>
+        <div className={cx(styles.centeredContent, styles.others)}>
+          <div className={cx(styles.containerBox, styles.centeredContent, styles.rowContainer)}>
+            <div className={cx(styles.headerSection, styles.headerSmall)}>Add a row</div>
+            <div className={cx(styles.bodySection, styles.bodySmall)}>
+              Group your visualizations into expandable sections.
+            </div>
+            <Button
+              icon="plus"
+              fill="outline"
+              aria-label="Add new row"
+              onClick={() => {
+                reportInteraction('Create new row');
+                onCreateNewRow();
+              }}
+              disabled={!canCreate}
+            >
+              Add row
+            </Button>
+          </div>
+          <div className={cx(styles.containerBox, styles.centeredContent, styles.libraryContainer)}>
+            <div className={cx(styles.headerSection, styles.headerSmall)}>Import panel</div>
+            <div className={cx(styles.bodySection, styles.bodySmall)}>
+              Import visualizations that are shared with other dashboards.
+            </div>
+            <Button
+              icon="plus"
+              fill="outline"
+              aria-label="Add new panel from panel library"
+              onClick={() => {
+                reportInteraction('Add a panel from the panel library');
+                onAddLibraryPanel();
+              }}
+              disabled={!canCreate}
+            >
+              Import library panel
+            </Button>
+          </div>
+        </div>
+      </div>
+    </div>
+  );
+};
+
+const getStyles = (theme: GrafanaTheme2) => {
+  return {
+    wrapper: css({
+      label: 'dashboard-empty-wrapper',
+      flexDirection: 'column',
+      maxWidth: '920px',
+      gap: theme.spacing.gridSize * 4,
+    }),
+    containerBox: css({
+      label: 'container-box',
+      flexDirection: 'column',
+      boxSizing: 'border-box',
+      border: '1px dashed rgba(110, 159, 255, 0.5)',
+    }),
+    centeredContent: css({
+      label: 'centered',
+      display: 'flex',
+      alignItems: 'center',
+      justifyContent: 'center',
+    }),
+    visualizationContainer: css({
+      label: 'visualization-container',
+      padding: theme.spacing.gridSize * 4,
+    }),
+    others: css({
+      label: 'others-wrapper',
+      alignItems: 'stretch',
+      flexDirection: 'row',
+      gap: theme.spacing.gridSize * 4,
+    }),
```

Comment:
```suggestion
    others: css({
      label: 'others-wrapper',
      alignItems: 'stretch',
      flexDirection: 'row',
      gap: theme.spacing.gridSize * 4,
      
      [theme.breakpoints.down('sm')]: {
        flexDirection: 'column',
      },
    }),
```

To fix some mobile styling 😄

Reply:
agree! better now